sonar-plugin-api/src/main/java/org/sonar/api/profiles/ProfileImporter.java

```diff
@@ -27,6 +27,8 @@
 import org.sonar.api.utils.ValidationMessages;
 
 /**
+ * Create a quality profile from an external configuration file.
```

Comment:
I don't find a better word, but external configuration file seems to vague. Rules file maybe?